loader/loader.go

```diff
@@ -193,8 +204,13 @@ func parseYAML(source []byte) (map[string]interface{}, PostProcessor, error) {
 	return converted.(map[string]interface{}), &processor, nil
 }
 
-// Load reads a ConfigDetails and returns a fully loaded configuration
+// Load deprecated, prefer LoadWithContext
```

Comment:
```suggestion
// Load reads a ConfigDetails and returns a fully loaded configuration.
//
// Deprecated: use LoadWithContext.
```

This should make it show up as deprecated in IDEs and hide it on `pkg.dev`